stackslib/src/net/p2p.rs

```diff
@@ -246,6 +246,7 @@ pub struct StacksTipInfo {
     pub consensus_hash: ConsensusHash,
     pub block_hash: BlockHeaderHash,
     pub height: u64,
+    pub coinbase_height: Option<u64>,
```

Comment:
The `coinbase_height` is always defined.  In Stacks 2.x, it's the same as the chain height since each block has a coinbase.  In Nakamoto, it's the number of coinbases that have occurred so far (since not every block has a coinbase).